### ISSUE TITLE
Remote debugging of Python function apps

### ISSUE THREAD

**Issue by felihong** [microsoft/vscode-azurefunctions#3630]
Hi there, 

is it possible to `remotely debug` a `Python function app` (premium or dedicated plan) in VS Code? 

In Azure portal I was able to enable remote debugging of my as shown below:
![Screenshot 2023-03-15 at 09 50 00](https://user-images.githubusercontent.com/28823328/225256427-0d37658a-c059-4188-b544-c42416ef9cfb.png)

When I try to start remote debugging using `VS Code Azure functions extension (v1.10.4)`, I got an error that 
```
Azure remote debugging is currently on supported for Node.js Function Apps running on Linux App Service plans. Consumption plans are not supported
``` 
Could you please confirm this and kindly give a hint whether there's any workarounds? Thank you!

**Comment by alexweininger** [comment]
We don't currently support remote debugging a Python function app. @MicroFish91 can you investigate this to see if we can add support?

**Comment by FaresKi** [comment]
I agree that this would be super helpful. I know there's an integrated debugging feature of .NET apps and Visual Studio, but for us VSCoders and Python folks, the possibility to remote debug would be super helpful.

**Comment by reynajavar** [comment]
Is this feature request still under investigation?

**Comment by Abdulaziz-eblacorp** [comment]
@MicroFish91 is this feature added 👀? 
if yes, where i can find it's docs ?

**Comment by nturinski** [comment]
Hi everyone, just wanted to give a small update.

I tried simply allowing the extension to try remote debugging a Python function app. While it was trying to create the tunnel proxy, there was no tunnel state returned.

I'm not sure if it's an error on the extension's end or if the service simply doesn't allow remote debugging for Python, but it's a little more complicated than me just turning it on for Python 😞 

I'll keep investigating, but if anyone has any documentation for remote debugging on a Python function app, I'd appreciate it if you could share it with me.
